gobblin-rest-service/gobblin-rest-api/build.gradle

```diff
@@ -112,6 +112,67 @@ artifacts {
     archives dataTemplateSourcesJar, dataTemplateJavadocJar, restClientSourcesJar, restClientJavadocJar
 }
 
+publishing {
+  publications {
+    dataTemplate(MavenPublication) {
+      //from components.java
+      artifactId dataTemplateName
+      artifacts = [artifact(tasks.mainDataTemplateJar), dataTemplateSourcesJar, dataTemplateJavadocJar]
+      pom pomAttributes
+      pom.withXml {
+        def dependenciesNode = asNode().appendNode('dependencies')
+
+        configurations.runtime.allDependencies.each {
```

Comment:
This seems to be duplicated in multiple build files. You can extract a method somewhere and use it to reduce the duplication, something like:

```
addRuntimeDependenciesToPom(pom)
```

To reuse build logic you can use the same techinque as with the existing "pomAttributes" method

Reply:
done